--- /Amethyst-Android/app_pojavlauncher/build.gradle
+++ /Amethyst-Android/app_pojavlauncher/build.gradle
@@ -86,10 +86,10 @@ android {
             keyPassword "android"
         }
         customRelease {
-            storeFile file("aamc_upload.jks")
-            storePassword System.getenv("GPLAY_KEYSTORE_PASSWORD")
-            keyAlias "android_github_release"
-            keyPassword System.getenv("GPLAY_KEYSTORE_PASSWORD")
+            storeFile file("keystore.jks")
+            storePassword System.getenv("STORE_PASSWORD")
+            keyAlias System.getenv("KEY_ALIAS")
+            keyPassword System.getenv("KEY_PASSWORD")
         }
         googlePlayBuild {
             storeFile file("upload.jks")
 
